pkg/controller/pcideviceclaim/pcideviceclaim_controller.go

```diff
@@ -343,6 +255,58 @@ func (h Handler) reconcilePCIDeviceClaims(name string, pdc *v1beta1.PCIDeviceCla
 	return pdc, nil
 }
 
+// This function adds the PCIDevice to the device plugin, or creates the device plugin if it doesn't exist
+func (h Handler) addToDevicePlugin(pd *v1beta1.PCIDevice, pdc *v1beta1.PCIDeviceClaim) error {
+	var err error
+	resourceName := pd.Status.ResourceName
+	logrus.Infof("[addToDevicePlugin] finding device plugin %s", resourceName)
+	dp, err := deviceplugins.Find(
```

Comment:
nit: the struct already references pointer values, may be should just check if the value is not nil, rather than returning a error?